dialect/sql/builder.go

```diff
@@ -2301,6 +2301,40 @@ func (s *Selector) AppendSelectExprAs(expr Querier, as string) *Selector {
 	return s
 }
 
+// PrependSelect prepends additional columns to the SELECT statement.
+func (s *Selector) PrependSelect(columns ...string) *Selector {
```

Comment:
Keep only those you use in the codegen. No need to add unused methods.

Reply:
Done